src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -96,23 +95,34 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
                 WebResponse response = await getResponse;
                 HttpWebResponse httpResponse = (HttpWebResponse)response;
                 httpResponse.Close();
-                Assert.Throws<ObjectDisposedException>(() =>
+                if (PlatformDetection.IsFullFramework)
                 {
-                    httpResponse.GetResponseStream();
-                });
+                    Stream stream = httpResponse.GetResponseStream();
+                }
+                else
+                {
+                    Assert.Throws<ObjectDisposedException>(() =>
+                    {
+                        httpResponse.GetResponseStream();
+                    });
+                }
             });
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17842")] // Hangs in Desktop
-        public async Task HttpWebResponse_Serialize_Fails()
+        public async Task HttpWebResponse_Serialize_ExpectedResult()
         {
             await LoopbackServer.CreateServerAsync(async (server, url) =>
             {
                 HttpWebRequest request = WebRequest.CreateHttp(url);
                 request.Method = HttpMethod.Get.Method;
                 Task<WebResponse> getResponse = request.GetResponseAsync();
-                await LoopbackServer.ReadRequestAndSendResponseAsync(server, "HTTP/1.1 200 OK\r\n");
+                DateTimeOffset utcNow = DateTimeOffset.UtcNow;
+                await LoopbackServer.ReadRequestAndSendResponseAsync(server,
+                        $"HTTP/1.1 200 OK\r\n" +
+                        $"Date: {utcNow:R}\r\n" +
```

Comment:
Yes.  It was causing a hang on Desktop.  I wasn't sure exactly why...but making it a correct HTTP response got rid of the problem.